snapshots/education/2023-07-17/education_barro_lee_projections.csv.dvc

```diff
@@ -0,0 +1,21 @@
+meta:
+  name: Datasets on Projections of Educational Attainment (Lee and Lee, 2015)
```

Comment:
would probably change the name? Having the word "Data Sets" in the name seems a bit redundant.

Maybe something like "Projections of Educational Attainment (Lee and Lee, 2015)".

(unless authors explicitly ask for this format      )